app/lib/theme/CustomDiffFields.tsx

```diff
@@ -20,193 +23,161 @@ const contentSuffixElement = (
   );
 };
 
-interface OverwriteNotificationArgument {
-  latestCommittedData: any;
-  isNumber?: boolean;
-  isDate?: boolean;
-  isMoney?: boolean;
-  isPercentage?: boolean;
-  numberOfDecimalPlaces?: number;
-}
+const renderData = (isDate: boolean | undefined, data: string | undefined) => {
```

Comment:
maybe this could be called `formatData`? At first I thought data was a typo for date